plugins/Makefile.mk

```diff
@@ -23,3 +23,8 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+dist_noinst_SCRIPTS += plugins/convert_README_to_header.sh
+
+plugins/%/PluginDescription.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
Ah I see.

I don't think it will have too much of a hit on maintenance. We've barely changed the other similar script based config in the Makefiles in the past. There's also the balance of it being obvious and in the expected place, rather than hidden in another file.